examples/n-body/rust/Cargo.toml

```diff
@@ -9,4 +9,4 @@ crate-type = ["cdylib"]
 
 [profile.release]
 lto = true
-opt-level = 3
```

Comment:
It should stay '3' instead 's' because AS also using `-O3` so for better comparison they should have same opt levels